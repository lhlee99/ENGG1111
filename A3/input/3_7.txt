5
Articuno -102.2 300.5 258.6
Zapdos -102.2 300.5 258.6
Moltres -102.2 300.5 258.6
Mewtwo -102.2 300.5 258.6
Mew -102.2 300.5 258.6
