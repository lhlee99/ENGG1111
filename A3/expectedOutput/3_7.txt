Articuno: Zapdos Moltres Mewtwo Mew
Zapdos: Articuno Moltres Mewtwo Mew
Moltres: Articuno Zapdos Mewtwo Mew
Mewtwo: Articuno Zapdos Moltres Mew
Mew: Articuno Zapdos Moltres Mewtwo
